Error on loading resource: /minting/annual-provisions

Hello!
trying out lunie light with a custom chain running. The npx proxy command is great to get around CORs headaches, i remember finding that gem when CORS was a constant danger. my chain is called [pool-toy](https://github.com/interchainberlin/pooltoy) and running v0.39.x. It has the following modules:
```
ModuleBasics    = module.NewBasicManager(
		genutil.AppModuleBasic{},
		auth.AppModuleBasic{},
		bank.AppModuleBasic{},
		staking.AppModuleBasic{},
		distr.AppModuleBasic{},
		params.AppModuleBasic{},
		slashing.AppModuleBasic{},
		supply.AppModuleBasic{},
		pooltoy.AppModuleBasic{},
		faucet.AppModule{},
	)
```
Lunie light is loading on my local dev and connecting to the correct endpoints but i see the following error before anything will render real content:
```
Failed to load resource: the server responded with a status of 404 (Not Found)
webpack-internal:///./apis/cosmos-source.js:171 Error for query minting/annual-provisions in network Pool Toy (tried 3 times)
_callee2$ @ webpack-internal:///./apis/cosmos-source.js:171
webpack-internal:///./node_modules/axios/lib/core/createError.js:16 Uncaught (in promise) Error: Request failed with status code 404
    at createError (webpack-internal:///./node_modules/axios/lib/core/createError.js:16)
    at settle (webpack-internal:///./node_modules/axios/lib/core/settle.js:17)
    at XMLHttpRequest.handleLoad (webpack-internal:///./node_modules/axios/lib/adapters/xhr.js:61)
```

--- a/apis/cosmos-reducers.js
+++ b/apis/cosmos-reducers.js
@@ -716,11 +716,13 @@ export function validatorReducer(
     maxChangeCommission: validator.commission.commission_rates.max_change_rate,
     status: statusInfo.status,
     statusDetailed: statusInfo.status_detailed,
-    expectedReturns: expectedRewardsPerToken(
-      validator,
-      validator.commission.commission_rates.rate,
-      annualProvision
-    ).toFixed(6),
+    expectedReturns: annualProvision
+      ? expectedRewardsPerToken(
+          validator,
+          validator.commission.commission_rates.rate,
+          annualProvision
+        ).toFixed(6)
+      : undefined,
   }
 }
 

--- a/apis/cosmos-source.js
+++ b/apis/cosmos-source.js
@@ -173,7 +173,7 @@ export default class CosmosAPI {
         this.query(`staking/validators?status=bonded`),
         this.query(`staking/validators?status=unbonded`),
       ]).then((validatorGroups) => [].concat(...validatorGroups)),
-      this.getAnnualProvision(),
+      this.getAnnualProvision().catch(() => undefined),
       this.getValidatorSet(height),
       this.getSignedBlockWindow(),
     ])

